src/test/groovy/com/github/blindpirate/gogradle/crossplatform/DefaultGoBinaryManagerTest.groovy

```diff
@@ -214,6 +214,18 @@ class DefaultGoBinaryManagerTest extends MockEnvironmentVariableSupport {
         verify(httpUtils).download(anyString(), any(Path))
     }
 
+    @Test
+    void 'go binary with specified version should be downloaded when a modified go binary base url has been set'() {
+        // given
+        when(setting.getGoVersion()).thenReturn('1.7.4')
+        when(setting.getgoBinaryDownloadBaseUri()).thenReturn(URI.create('http://example.com/'))
```

Comment:
I believe you have a tiny typo here:

`setting.getgoBinaryDownloadBaseUri()`->`setting.getGoBinaryDownloadBaseUri()`